engine/src/main/resources/swagger/cromwell.yaml

```diff
@@ -437,6 +447,7 @@ paths:
         '500':
           $ref: '#/responses/ServerError'
     post:
+      operationId: query
```

Comment:
Overloading `query` works in Scala (and I'd guess Java) because of the different input types, but not sure it'd be safe for all languages that might try to generate a client.